proof/invariant-abstract/AARCH64/ArchKHeap_AI.thy

```diff
@@ -828,6 +828,34 @@ lemma valid_arch_state_lift_aobj_at:
 end
 end
 
+\<comment> \<open>FIXME: if tcb_at and related terms were moved to InvariantsPre_AI then this lemma could go in
+     ArchInvariants_AI.\<close>
```

Comment:
you might want to make this and the one below a `FIXME arch-split`, I'm not sure right now